uiowa/uiowa
# Grad admissions site split

Closes #4521 
Closes #4475 
Closes #4476 
Closes #4568


# How to test



1. checkout branch
2. blt ds --site=grad.admissions.uiowa.edu
3. composer install
3. drush @admissionsgrad.local en tvi
3. drush @admissionsgrad.local en grad_admissions_core
4. drush -l grad.admissions.uiowa.edu config:import --source ../config/sites/grad.admissions.uiowa.edu --partial
5. drush @admissionsgrad.local cim
7. blt frontend
8. . drush @admissionsgrad.local cr
6. drush @admissionsgrad.local uli
7. Add an estimated cost (http://admissionsgrad.uiowa.ddev.site/node/add/costs_estimate) and click "US Citizens & Permanent Residents". Select ‘provide a menu link’ -> Select ‘Cost and Funding’ -> Select ‘Accounting’ disabled link I added to prod. Save.
8. Add another estimated cost, follow same steps except select ‘International’. Save
9. Add a college. Go to http://admissionsgrad.uiowa.ddev.site/admin/structure/taxonomy/manage/colleges/add
8. Add an area of study (http://admissionsgrad.uiowa.ddev.site/node/add/area_of_study) and add an area of study (ex accounting) a program (Accounting (MAc), reference college you created, an estimated cost intro and reference estimated cost you created (SHOULD ONLY DISPLAY DOMESTIC - Show up as Accounting, Estimated Cost or whatever title you added)
9. Make sure the college displays as a link on aos. Click link and make sure college page displays the aos that referenced it and looks purty .
10. If college looks decent, go back to the aos page you created and make sure the estimated cost you references shows up as a purty button -> click said button and make sure it takes you to domestic cost page.
11. P.S. Also make sure there's a tabbed menu that shows the international estimated cost page you added and can switch over to it
10. Add another area of study and make sure can add same estimated cost
